content/docs/tutorials/aws/ecs-fargate.md

```diff
@@ -1,149 +1,194 @@
 ---
-title: "Containers on ECS \"Fargate\""
+title: "Containers on ECS Fargate"
 
 aliases: ["/docs/reference/tutorials/aws/tutorial-service/"]
 ---
 
-In this tutorial, we'll build and publish a Docker container image to a private Elastic Container Registry (ECR), and
-spin up a load balanced Amazon Elastic Container Service (ECS) "Fargate" service, all in a handful of lines of code,
-using [Pulumi Crosswalk for AWS]({{< relref "/docs/guides/crosswalk/aws" >}}).
+<!--To-Do: Update github-buttons shortcode to check for existence of examples-->
+<p class="mb-4 flex">
+    <button class="flex flex-wrap items-center rounded text-xs text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white h-8" href="https://github.com/pulumi/examples/tree/master/aws-ts-containers" target="_blank">
```

Comment:
Clicking the button doesn't do anything. It needs to be an `<a>` not a `<button>`.

Reply:
Done.